spec/attribute.dd

```diff
@@ -508,9 +508,7 @@ $(H2 $(LNAME2 inout, $(D inout) Attribute))
 
 $(H2 $(LNAME2 shared, $(D shared) Attribute))
 
-        $(P The $(DDSUBLINK spec/const3, shared, $(D shared) attribute) modifies the type from $(D T) to $(D shared(T)),
-        the same way as $(D const) does.
-        )
+        $(P See $(DDSUBLINK spec/const3, shared, $(D shared)).)
```

Comment:
I edited this as shared also affects global variable storage, not just their type.